src/launcher/launcher_main.cpp

```diff
@@ -378,6 +378,17 @@ native_string produce_mod_path() {
 	return simple_fs::extract_state(dummy);
 }
 
+void save_playername() {
+	auto path = launcher::produce_mod_path();
+	auto game_state = std::make_unique<sys::state>();
```

Comment:
No, this cannot be done, this is several hundred megabytes of stuff you are spinning up just to save a string to a file...